pom.xml

```diff
@@ -26,6 +26,7 @@
     <module>jr-stree</module>
     <module>jr-annotation-support</module>
     <module>jr-all</module>
+    <module>jr-extension-datetime</module>
```

Comment:
Let's move this above `jr-all` (since it needs to be built first: Maven can figure it out but just in case)